src/main/java/no/unit/nva/model/contexttypes/utils/IssnUtil.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.model.contexttypes.utils;
+
+import no.unit.nva.model.exceptions.InvalidIssnException;
+import no.unit.nva.model.validator.IssnValidator;
+
+import static java.util.Objects.isNull;
+
+public class IssnUtil {
```

Comment:
This should be a final class. I see that Jan has disabled the rule [UtilityClass](https://pmd.github.io/latest/pmd_rules_java_design.html#useutilityclass), I do not know why. But this class cannot be extended.